packages/gatsby/src/query/index.ts

```diff
@@ -58,9 +58,16 @@ export { calcDirtyQueryIds as calcInitialDirtyQueryIds }
  * Groups queryIds by whether they are static or page queries.
  */
 export function groupQueryIds(queryIds: Array<string>): IGroupedQueryIds {
-  const grouped = _.groupBy(queryIds, p =>
-    p.slice(0, 4) === `sq--` ? `static` : `page`
-  )
+  const grouped = _.groupBy(queryIds, p => {
+    const prefix = p.slice(0, 4)
+    if (prefix === `sq--`) {
+      return `static`
+    } else if (prefix === `fr--`) {
```

Comment:
Yes, I can replace it with something else because that's internal only - annoyance is that "slice" also start with an "s" (like static query) so it might be confusing unless I use full name

Reply:
`sq` vs `sl` could be hard to spot the diff. `sq` vs `slice` looks better

Reply:
Done in [`32765cf` (#36489)](https://github.com/gatsbyjs/gatsby/pull/36489/commits/32765cf26c5dc688bfc9a04c8be0b3da1edf491b)